admin_manual/release_notes.rst

```diff
@@ -191,6 +192,8 @@ Known issues
 ~~~~~~~~~~~~
 
 - `The new Password Policy feature "Password Expiration" <https://doc.owncloud.com/server/latest/admin_manual/release_notes.html#the-password-policy-extension>`_ does not work together with Multi-Factor Authentication (e.g. ``twofactor_totp``, ``twofactor_privacyidea``). Please do not deploy expiration policies yet when having Two- or Multi-Factor Authentication extensions in place. This issue will be solved with the next ownCloud Server release. See `#32059 <https://github.com/owncloud/core/issues/32059>`_ for more information.
+- `The new Password Policy feature "Password Expiration" <https://doc.owncloud.com/server/latest/admin_manual/release_notes.html#the-password-policy-extension>`_ includes an 
+*occ* command to manually force password expiration which is necessary to use when imposing expiration policies on an instance with existing users. Currently the command will only work when the policy *X days until user password expires* has been enabled. This might be confusing and will be solved with the next release of the extension. See `#66 <https://github.com/owncloud/password_policy/issues/66>`_ for more information.
```

Comment:
- "expiration which" => "expiration. Please run it directly after imposing ..." (call to action :-D)